examples/example.py

```diff
@@ -18,11 +21,11 @@ def main():
     os.system("cls")

```

Comment:
Add this to the `Output` class and then to `TerminalOutput`.

Reply:
i added it, should i test it? if so how?

Reply:
Maybe run something like this?
`python mycliprogram --someoption "some input" | diff correct_output -`